test-common/uk/gov/hmrc/apiplatform/modules/submissions/SubmissionsTestData.scala

```diff
@@ -32,28 +32,28 @@ trait StatusTestDataHelper {
 
     def hasCompletelyAnsweredWith(answers: Submission.AnswersToQuestions): Submission = {
       (
-        Submission.addStatusHistory(Submission.Status.Answering(DateTime.now.withZone(DateTimeZone.UTC), true)) andThen
+        Submission.addStatusHistory(Submission.Status.Answering(LocalDateTime.now(ZoneOffset.UTC), true)) andThen
```

Comment:
There is a trait ClockNow that allows for now with offset and truncation to millis.  Might be worth a think. Also has a fixedclock trait for testing.

Reply:
Left as system default deliberately to display date/times (although we're only displaying dates here so it's not going to make any difference).